internal/pkg/skuba/cni/cilium_test.go

```diff
@@ -172,9 +172,13 @@ func Test_AnnotateCiliumDaemonsetWithCurrentTimestamp(t *testing.T) {
 	for _, tt := range tests {
 		tt := tt // Parallel testing
 		t.Run(tt.name, func(t *testing.T) {
-			tt.clientset.AppsV1().DaemonSets(metav1.NamespaceSystem).Create(tt.daemonset)
+			_, err := tt.clientset.AppsV1().DaemonSets(metav1.NamespaceSystem).Create(tt.daemonset)
+			if err != nil && !tt.errExpected {
```

Comment:
This is a little weird, we are using the `fake.Clientset` here. Is there any benefit in checking this here? Can we make linter and errcheck ignore certain blocks? I'm fine adding this if it cannot be skipped, but looks a little cumbersome to add all this logic in tests. [1]

Reply:
it can be skipped and I think it should :upside_down_face: